float64 sensorReading
---
float64 filteredOutput
